downstream/modules/automation-hub/con-rh-certified-synclist.adoc

```diff
@@ -1,9 +1,15 @@
 [id="con-rh-certified-synclist"]
 
-= About Red Hat Certified collections synclists
+= About Red Hat {CertifiedName} synclists
 
-A synclist is a curated group of Red Hat Certified collections that is assembled by your organization administrator that syncs to your local {HubName}. 
-Use synclists to manage only the content that you want and exclude unnecessary collections.
-You can design and manage your synclist from the content available as part of Red Hat Certified collections on console.redhat.com
+A synclist is a curated group of Red Hat Certified collections that is assembled by your organization administrator that synchronizes with your local {HubNameMain}. 
+Using synclists enables you to manage only the content that you want and exclude unnecessary collections.
```

Comment:
Suggest changing to:
You can use synclists to manage only the content that you want and exclude unnecessary collections.